packages/asset/contracts/Asset.sol

```diff
@@ -332,15 +244,6 @@ contract Asset is
         }
```

Comment:
_setMetadataHash blocks a creator from reusing the same metadataHash for multiple tokens for a given creator, when it should allow a new tokenId to be made?

Suggest also changing `"metadata hash mismatch for tokenId"` to `"metadata hash mismatch for existing tokenId"`

Reply:
Yes, remember we discussed that we will include the timestamp in the metadata hash so every time a new asset is created it will have an unique metadata hash.

Probably we don't even need that check then.